### ISSUE TITLE
how to check fitting performance

### ISSUE THREAD

**Issue by ramdhan1989** [kdgutier/esrnn_torch#19]
is there any way to check whether the model is overfit visually ? need advice to avoid overfit model using this repo.

thank you

**Comment by kdgutier** [comment]
Hi @ramdhan1989, sorry for the late response.
For the moment we don't have a visualization of the overfit metrics but we added evaluation during training to the model that you can plot.

As a quick advice there is a very strong regularization technique for regression problems called "early stopping" that you can track if you plot the validation loss against the gradient steps to decide where to cut the training. 

Yao, Y., Rosasco, L., & Andrea, C. (2007). On early stopping in gradient descent learning. Constructive
Approximation, 26(2), 289{315. URL: https://doi.org/10.1007/s00365-006-0663-2.
